docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,116 @@
+
+The iGPU 780M of AMD Ryzen CPU could run on ROCm at Linux with extral settings. 
+
+Available CPU: Ryzen 7000s/8000s CPU with iGPU 780M
+
+## Test Platform
+
+| **Platform ** | **miniPC**                                                      |
+| ------------- | --------------------------------------------------------------- |
+| **HW**        | AMD Ryzen R8845HS + Radeon780M(iGPU, set 16GB VRAM of 64GB DDR) |
+| **OS**        | Ubuntu22.04                                                     |
+| **SW**        | ROCm6.0+PyTorch                                                 |
+
+
+## Prerequisites
+0. Set UMA for iGPU in BIOS.
+1. Install GPU Driver and ROCm
```

Comment:
I think just the driver should be required.  ROCm should be optional.

Reply:
It is OK for me.